src/index.js

```diff
@@ -6,36 +6,83 @@ import Game from './Game'
 import { createStore } from 'redux'
 import './index.css';
 
-Mousetrap.bind('up', () => store.dispatch({type: 'ROTATE'}))
-Mousetrap.bind('left', () => store.dispatch({type: 'LEFT'}))
-Mousetrap.bind('right', () => store.dispatch({type: 'RIGHT'}))
+let pausedBoolean = false
 
-const reducer = ( state = new Game(), action ) => {
-  switch (action.type) {
+const dispatchTick = () => {
+  setTimeout( () => store.dispatch({ type: 'TICK' }), 400 )
+}
+
+const isGameInactive = ( state ) => {
+  return ( pausedBoolean === true ||
+    (state.game1.isGameOver() === true || state.game2.isGameOver() === true ) )
+}
+
+const gameStream = ( state = { 'game1': new Game(), 'game2': new Game() }, action ) => {
+  switch ( action.type ) {
     case 'TICK':
-      const revedState = state.tick()
-      if ( !revedState.isGameOver() || !revedState.isPaused() ) {
-        setTimeout( () => store.dispatch({ type: 'TICK' }), revedState.interval )
+      const revedState = {'game1': state.game1.tick(), 'game2': state.game2.tick() }
+      if ( !revedState.game1.isGameOver() &&
+      !revedState.game2.isGameOver() &&
+      pausedBoolean === false ) {
+        dispatchTick()
       }
       return revedState
     case 'ROTATE':
-      return state.rotate()
+      return isGameInactive( state ) ?
```

Comment:
When using redux, you typically want to avoid mutating the state object, instead returning a *new*, updated state copy.  There's a few reasons, but one is efficiency - here's a [few](https://github.com/reactjs/redux/issues/758), [links](http://redux.js.org/docs/introduction/ThreePrinciples.html).

I would suggest changing this (and other locations this is done) to:
```js
return isGameInactive( state ) ? state : Object.assign( {}, state, { game1: state.game1.left() } )
```